h2o-algos/src/test/java/hex/grid/GridSearchWithFilteringTest.java

```diff
@@ -1,18 +1,18 @@
 package hex.grid;
 
-import hex.grid.filter.KeepOnlyFirstMatchFilterFunction;
+import hex.grid.filter.DefaultPermutationFilter;
+import hex.grid.filter.PermutationFilter;
```

Comment:
btw, mathematically speaking, these are combinations, not permutations.

Reply:
@seb-h2o agree, but it was a legacy naming and it makes sense to keep names that people are used to. Another concern is that word `combination` feels too generic as it is overused in so many different contexts.  I had `gridItem` naming before to make it quite specific. Do you suggest to change it to combination?

Reply:
if it's legacy naming, please keep